docs/en/observability/uptime-intro.asciidoc

```diff
@@ -13,6 +13,8 @@ infrastructure with {heartbeat} natively.
 For browser-based monitors, a richer management and reporting experience,
 and more capabilities such as triaging and responding to alerts, use the
 <<monitor-uptime-synthetics,{synthetics-app}>> instead of the {uptime-app}.
+
+Note that {uptime-app} is no longer shown by default if there is no recent Heartbeat data. If you do not see {uptime-app} in the Observability menu, this will need to be enabled. This can be done under Kibana Advanced Settings by switching on _Always show legacy Uptime app_ (`observability:enableLegacyUptimeApp`).
```

Comment:
@paulb-elastic Quick question: Should the setting (`observability:enableLegacyUptimeApp`) also be documented in the [Advanced Settings](https://www.elastic.co/guide/en/kibana/master/advanced-options.html) topic in the Kibana docs?

Reply:
@dedemorton yes, indeed - this is an oversight on my part, I didn't know that existed.  I'll create a Kibana PR for those docs.  Thanks for the steer.  I'll mention the PR here once it's created.

Reply:
https://github.com/elastic/kibana/pull/183613 raised